src/components/FilterSearch.tsx

```diff
@@ -109,6 +109,22 @@ export function FilterSearch({
     }
   }, [answersActions, currentFilter, searchOnSelect]);
 
+  const handleBlurDropdown = useCallback((_prevValue, _value, _index, itemData) => {
+    const newFilter = itemData?.filter as Filter;
+    const newDisplayName = itemData?.displayName as string;
+    if (newFilter && newDisplayName) {
+      if (currentFilter) {
+        answersActions.setFilterOption({ ...currentFilter, selected: false });
+      }
+      answersActions.setFilterOption({ ...newFilter, displayName: newDisplayName, selected: true });
+      setCurrentFilter(newFilter);
+      answersActions.setOffset(0);
+      if (searchOnSelect) {
```

Comment:
Do we still want this behavior to depend on `searchOnSelect` or should it depend on a new prop?

Reply:
You're right, I'll remove that conditional. I don't think it would make sense for a search to be triggered anytime the dropdown is blurred, so I'm not sure if another prop is needed.